src/Tabs/StyledTab.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import Flex from '../Flex';
+
+const activeStyles = ({ active, theme }) => {
+  const color = theme.colors.intent.brand.dark;
+
+  return (
+    active &&
+    css`
+      color: ${color};
+      border-color: ${color};
+
+      &:focus,
+      &:hover {
+        background: transparent;
+      }
+    `
+  );
+};
+
+const Tab = styled(Flex)`
```

Comment:
Should this be called `StyledTab` here instead?

Reply:
Good catch